knowledge_storm/rm.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import os
-from typing import Callable, Union, List
+from typing import Callable, Union, List, Dict
+from typing_extensions import Dict
```

Comment:
Was originally struggling with figuring out the best way to type query_params since it can be both a Dict as well as a list of Dict depending on what the user passes in. I've removed it and added more clarification in the rm.py file for serper.

Reply:
I see. Could we type it as `Union[Dict, List[Dict]]`?